sales_report_product_image/__manifest__.py

```diff
@@ -5,13 +5,16 @@
     'name': 'Product Image for Sale',
     'author': 'Serpent Consulting Services Pvt. Ltd.',
     'maintainer': 'Serpent Consulting Services Pvt. Ltd.',
+    'version': '11.0.1.0.0',
     'category': 'Sales Management',
     'summary': 'Product Image for Sale Reports',
     'license': 'AGPL-3',
     'website': 'http://www.serpentcs.com',
-    'version': '10.0.1.0.0',
-    'sequence': 1,
-    'depends': ['sale', 'web_tree_image'],
```

Comment:
We have to add the repo on oca _dependeciex.text?

Reply:
Yes we will need to but the web_tree_image module is not migrated yet in v11 in OCA web.